samples/nrf9160/gps/src/main.c

```diff
@@ -35,7 +35,17 @@
 #elif CONFIG_GPS_SAMPLE_ANTENNA_EXTERNAL
 #define AT_COEX0       "AT\%XCOEX0"
 #endif
+#endif /* CONFIG_BOARD_NRF9160DK_NRF9160NS */
+
+#ifdef CONFIG_BOARD_THINGY91_NRF9160NS
+#define AT_MAGPIO      "AT\%XMAGPIO=1,1,1,7,1,746,803,2,698,748,2,1710,2200," \
+			"3,824,894,4,880,960,5,791,849,7,1565,1586"
+#ifdef CONFIG_GPS_SAMPLE_ANTENNA_ONBOARD
+#define AT_COEX0       "AT\%XCOEX0=1,1,1570,1580"
```

Comment:
Is the frequency range intentionally different here than in that Kconfig option default and `AT%MAGPIO` above? Seeing https://github.com/nrfconnect/sdk-nrf/commit/e522e4f426148e9355ac66c5e573d1fbfb6a3e24 I'm a bit in doubt.

Reply:
Updated to the range that new modems are certified for